lib/index.js

```diff
@@ -54,20 +65,23 @@ class PWMetrics {
           results[index] = data;
         }).catch(error => {
           console.error(getMessageWithPrefix('ERROR', 'FAILED_RUN', index, results.length, error.message));
-          throw new Error(error);
+          console.error(error.stack + '\n');
```

Comment:
Because it would send an error to https://github.com/pedro93/pwmetrics/blob/145b489f98636a5d37c7f2e5fe8523deed965f38/lib/index.js#L74 which does not have a catch clause. Suppose the run 4/10 throws, by throwing the error, the remaining runs are not completed:

With the throw Error I had to manually cancel the process (gave it 5m to recover).
![screen shot 2017-03-16 at 09 19 25](https://cloud.githubusercontent.com/assets/3718654/23989243/b191451a-0a29-11e7-88f9-3f5d71423f98.png)